utils/wasm-builder/src/wasm_project.rs

```diff
@@ -34,6 +34,15 @@ use fs2::FileExt;
 
 use itertools::Itertools;
 
+/// Print an info message.
+fn print_info_message(message: &str) -> String {
```

Comment:
That doesn't seem to be printing anything. Consider either renaming this method or adding format args and make it actually handling printing.

Reply:
Ah, I noticed this method is used for formating an message for an `Err`. Then I guess renaming is the least hassle.

How about, idk, `colorize`?

Reply:
Done